### DISCUSSION TITLE
[blocks]: Expandable sidebar

### DISCUSSION THREAD

**Discussion by andersontz** [shadcn-ui/ui#3539] [Blocks Request]
### Description

An example of a dashboard, like dashboard-05 for example, but with the possibility of expanding or shrinking the sidebar, and also with the possibility of changing the theme cores according to the pre-defined cores that we already have on the home page of your website . And multilevels menus of sidebar, with 3 leves.

**Comment by ishaanw007** [comment]
https://github.com/shadcn-ui/ui/discussions/3652
please check I have implemented it
